Lib/_pyrepl/historical_reader.py

```diff
@@ -71,6 +71,18 @@ def do(self) -> None:
         r.select_item(r.historyi - 1)
 
 
+class history_search_backward(commands.Command):
+    def do(self) -> None:
+        r = self.reader
+        r.search_next(False)
```

Comment:
Suggestion: Using keyword arguments for booleans makes it easier to read the code without looking at the definition of search_next:

```suggestion
        r.search_next(forwards=False)
```